### PR TITLE
Release notes 2019.1

### PR DESCRIPTION
**Pull request by joentje-manu**

### CODE REVIEW

File: website/blog/2019-03-19-2019_1.md [hunk 1]
```diff
@@ -92,11 +92,14 @@ A task is added to rename a process action.
...
 
 ### Sorted look-up combo's
-Look-up control can now be set to 'Combo (sorted)'. This will present the values in the combo using the sorting of the look-up table. The look-up control type 'Combo' has been renamed to 'Combo (alphabetical)' to reflect the difference between the two combos.
+Look-up control can now be set to *Combo (sorted)*. This will present the values in the combo using the sorting of the look-up table. The look-up control type *Combo* has been renamed to *Combo (alphabetical)* to reflect the difference between the two combos.
 
 ### Card list translation
 The translations have been extended with translations for the card list.
 
+### Approve validation messages
+It is now possible to set if a validation may be approved or not. By default it is not possible to approve errors, bus this can be changed. This also goes the other way around. By default is is possible to approve information messages or warnings, bus this can also be changed.
```
Comment (jsprklst) [inline website/blog/2019-03-19-2019_1.md]: is is, bus

**Comment by esprik** [comment]
"In some cases this will lead to a warning (not all validations are solved), and in some cases this will lead to not being able to execute the synchronization (no roles are present in the project version).". Repeating the same parts. Maybe "In some cases, this will lead to a warning (for example, not all validations are solved), and in other cases, it won't even be possible to execute the synchronization (for example, no roles are present in the project version)."

**Comment by esprik** [comment]
"This will lead to the project version being generated, to be synchronized to IAM, and the creation of the deployment package."

After execution, the following actions will be executed:

- the project version will be generated, 
- the project version will be validated,
- the source code will be created for the install (full) and for the upgrade (smart), 
- a synchronization script will be created, 
- a manifest will be made to use in the deployment center,
- and all icons and reports that are in the project folder will be copied.

**Comment by esprik** [comment]
History:
Add a line about unlocking a failed execution run?

**Comment by esprik** [comment]
"Does not apply."

"There are no new features implemented in this release."?

**Comment by esprik** [comment]
Why did you add (IAM) of (SF) to every subheader?

**Comment by esprik** [comment]
Changed (IAM)

There are some changes, could you describe them?
- Synchronization is removed
- Roles configuration is removed
- 2 rights tab pages are removed

**Comment by esprik** [comment]
Fixed (IAM)

Very technical. Can we rewrite this so we talk about the features that did not work and now do?

**Comment by esprik** [comment]
"Some scenarios of reordinging using using drag-drop and multi-selection did not work properly"

Reordinging?
using using?

**Comment by esprik** [comment]
Can we shorten the titles under Software Factory -> New. For example, the first title is almost the same as the block below. We could call this "Renumber tasks".

**Pull request merged**